lib/src/widgets/raw_animated_snack_bar.dart

```diff
@@ -101,9 +104,10 @@ class RawAnimatedSnackBarState extends State<RawAnimatedSnackBar> {
     } else {
       if (widget.mobileSnackBarPosition == MobileSnackBarPosition.top) {
         if (isVisible) {
-          return 70 + widget.getInitialDy();
+          return (widget.mobilePositionSettings?.topOnAppearance ?? 70) +
```

Comment:
Isn't it better to use a const object in the constructor to add default values and not null checking them here, Like what I did for `snackBarStrategy` for example?

Reply:
Yes, i think it would be better, ill fix it.